app/services/blog_service.rb

```
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+require 'yaml'
```

RuboCop (1.73)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)